gremlinapi/http_clients.py

```diff
@@ -148,37 +154,44 @@ def api_call(cls, method, endpoint, *args, **kwargs):
 class GremlinAPIurllibClient(GremlinAPIHttpClient):
```

Comment:
Refactor to replace Union[object, object] in all files that consume these classes